etc/ReplayOfflineConfiguration.py

```diff
@@ -35,8 +37,19 @@
 
 # Settings up sites
 processingSite = "T0_CH_CERN"
```

Comment:
Hi @germanfgv , I believe this should be T2_CH_CERN

Reply:
You are right! I fixed it